src/main/resources/application-preprod.yaml

```diff
@@ -71,9 +71,9 @@ SECURITYTOKENSERVICE_URL: https://sts-q2.preprod.local/SecurityTokenServiceProvi
 AKTOERID_URL: https://app-q2.adeo.no/aktoerregister
 PERSON_V3_URL: https://app-q2.adeo.no/tpsws-aura/ws/Person/v3
 ORGANISASJON_V5_URL: https://modapp-q1.adeo.no/ereg/ws/OrganisasjonService/v5
-DOKARKIV_V1_URL: http://dokarkiv.q2
```

Comment:
Har de gått bort fra q1 og q2? Trodde vi flyttet til q2 for å få prod-data? @hensol

Reply:
Nei,
"vi tok å fjernet appen i q2 namespace på fredag. Bruk ingressene over eller http://dokarkiv.teamdokumenthandtering"